tests/integration/iop-integration-test-defaults.yaml

```diff
@@ -4,6 +4,9 @@ kind: IstioOperator
 spec:
   meshConfig:
     accessLogFile: "/dev/stdout"
+    defaultConfig:
+      proxyMetadata:
+        ISTIO_META_DNS_CAPTURE: "ALL"
```

Comment:
should we disable prior to merge?

Reply:
No. The whole point of this effort was to make sure that DNS interception works smoothly on a pod and VM. At some point, we want this to be the default. In 1.7, this can be optional [but atleast fully tested, unlike the istio agent dns] but once we have full confidence that this works without any issues on k8s and VMs, I see no reason to keep it off. It will simplify a lot of things in multicluster, service entries, etc.